examples/kubernetes-istio/kafka-v1-policy.yaml

```diff
@@ -3,6 +3,13 @@ kind: CiliumNetworkPolicy
 metadata:
   name: kafka-authaudit
 specs:
+  - endpointSelector:
+      matchLabels:
+        app: kafka
+    ingress:
+    - fromEndpoints:
```

Comment:
This is not wrong but you can add this to the existing endpointSelector if you want. So you just add the following below the existing `ingress:`

``` yaml
    - fromEndpoints:
      - matchLabels:
          app: kafka
```

Reply:
will do